docs/docs/gatsby-core-philosophy.md

```diff
@@ -2,7 +2,177 @@
+
+- _cheap, secure, and maintainable by default_. Built Gatsby sites are just files living on a CDN. This reduces risk by removing the CMS as an attack surface, drastically reduces cost to serve your site at scale, and reduces the need to keep running servers.
+
+### Progressive disclosure of complexity
+
+In user experience, there’s a concept from the Nielsen Norman Group called “progressive disclosure", which is an approach to designing better user interfaces (UIs).
+
+From their summary:
+
+> Progressive disclosure defers advanced or rarely used features to a secondary screen, making applications easier to learn and less error-prone.
+
+A common use of this pattern is on user profiles. If you’ve ever seen a page that has a link to “advanced settings”, that’s progressive disclosure: most people don’t care about or need to touch the advanced settings, so those are moved to a secondary screen where only power users will bother to go. This simplifies the experience for most people without limiting the abilities of more advanced users.
+
+What’s powerful about this approach is that it gives the UI the ability to “react” to feedback. An advanced learner can ask for more, and the “teacher” (the UI) can turn up the complexity to meet the learner’s needs.
+
+#### Abstractions
+
+The most important concept of progressive disclosure is abstraction.
+
+Put simply, abstractions are a set of default decisions that allow someone to accomplish something while remaining blissfully unaware that any decisions were made at all. When done well, good abstractions are how we make the right thing the easy thing.
+
+If someone wants to change the defaults, they opt out of the abstraction and make those decisions on their own — and this is where things get tricky. To make sure we have progressive disclosure of complexity, we need to avoid a scenario where the abstraction becomes all-or-nothing.
+
+#### How this applies to Gatsby
+
+One of Gatsby's key roles is to bundle modern Javascript build tools. These include React, Babel, webpack, react-hot-loader, and so on.
+
+These systems are incredibly powerful. At the same time, they are quite complex. Users shouldn't _need_ to know these systems exist.
+
+Instead, their ES6 code should seamlessly transpile into a form accessible by modern browsers. If they want to use Typescript or Reason, they should be able to. If they want to upgrade (or downgrade!) from specific versions of React, they should be able to.
+
+Some "zero-config" frameworks bundle complexity together that is only configurable by "ejecting". The challenge with ejecting is that it takes you from simple to complex as soon as you move out of the happy path.
+
+## Gatsby's community philosophy
+
+### Work in the open
+
+Open source is at the core of Gatsby’s success, and one of the central tenets of open source is that things are done in the open and without smoke and mirrors.
+
+One of Gatsby’s core strengths are the strength of its community and ecosystem.
```

Comment:
```suggestion
Two of Gatsby’s core strengths are its community and ecosystem.
```
Or if you were trying to form it in the sentence as a single strength, it wasn't adding up (literally) lol.